concepts/headers-includes/about.md

```diff
@@ -0,0 +1,92 @@
+# About
+
+C requires that all functions are declared _before_ they are defined.
+This can be done at the top of the `*.c` file containing the definition, but there are advantages in splitting declarations into a separate header file with a `.h` extension.
+
+You can think of the header file as an API.
+The header will tell you _what_ a codebase has to offer, without going into the details of _how_.
+
+## Header files
+
+A header file can range from just a few lines to several hundred.
+
+For a simple example, we can look at the `Hello World` exercise (which you already completed). This is the [`hello_world.h` file][hello-h] which is part of that exercise:
+
+```c
+// This is called an include guard, which ensures that the header is only
+// included once. You could alternatively use '#pragma once'. See
+// https://en.wikipedia.org/wiki/Include_guard.
+#ifndef HELLO_WORLD_H
+#define HELLO_WORLD_H
+
+// Declare the 'hello()' function, which takes no arguments and returns a
+// 'const char *', i.e. a pointer to a character (in this case the first
+// character in a string). The function itself is defined in the hello_world.c
+// source file. This function is called by the test case(s) in the test source
+// file test_hello_world.c.
+const char *hello(void);
+
+#endif
+```
+
+Most of it is explanatory comments (lines starting with `//`).
+
+### Declarations
```

Comment:
I'm not sure about this whole section. It talks about parameters but not about the name or return type.
If this is the main explanation of function declarations I would like it to be more extensive.
But is this really the place for that? I'd prefer a separate concept for functions that contains function declarations.